### DISCUSSION TITLE
这个问题有没有人帮忙解决下

### DISCUSSION THREAD

**Discussion by gfl13453001** [vuejs/vitepress#1390] [Q&A] [answered]
(node:4260) UnhandledPromiseRejectionWarning: file:///F:/demo/noteV2.0/node_modules/vitepress/dist/node/serve-38d4ddb1.js:1
import { setDefaultResultOrder } from 'node:dns';
         ^^^^^^^^^^^^^^^^^^^^^
SyntaxError: The requested module 'node:dns' does not provide an export named 'setDefaultResultOrder'
    at ModuleJob._instantiate (internal/modules/esm/module_job.js:104:21)
    at async ModuleJob.run (internal/modules/esm/module_job.js:149:5)
    at async Loader.import (internal/modules/esm/loader.js:166:24)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:4260) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:4260) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by gfl13453001** [comment]
启动vitepress时报错

**Comment by brc-dd** [comment, marked as answer]
Duplicate of #1384  (upgrade your node.js version)

**Reply by gfl13453001** [reply]
感谢
